tests/test_bots.py

```diff
@@ -0,0 +1,46 @@
+#-*- encoding: utf-8 -*-
+
+
+import pytest
+import unittest
+
+try:
+    from mock import Mock, call, patch, ANY
+except ImportError:
+    from unittest.mock import Mock, call, patch, ANY
+
+from olclient.openlibrary import OpenLibrary
+from olclient.bots import BaseBot
```

Comment:
This is causing the following error and I don't know why
```
============================================ test session starts =============================================
platform linux -- Python 3.5.2, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: <truncated>/openlibrary-client, inifile: setup.cfg
collected 36 items / 1 error / 35 selected                                                                   

=================================================== ERRORS ===================================================
____________________________________ ERROR collecting tests/test_bots.py _____________________________________
ImportError while importing test module '/home/guyrandy/Documents/Repositories/openlibrary-client/tests/test_bots.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_bots.py:14: in <module>
    from olclient.bots import BaseBot
E   ImportError: No module named 'olclient.bots'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================== 1 error in 0.73s ==============================================
```

Reply:
My guess would be add `from .bots import BaseBot` to `olclient/__init__.py`.
If that does not work then ping me on Slack and I will dig in with you.

Reply:
Still getting this error. Sent you a message on slack.

Reply:
For some strange reason adding an `__init__.py` file to the test directory fixed this problem. I got the solution from [stackoverflow](https://stackoverflow.com/a/41752043) but it appears whether or not the test directory needs an `__init__.py` depends on the phase of the moon...

I'll continue to write tests for the time being but I suspect the root cause is somewhere else.

Reply:
Agreed...  I also like to keep `__init__.py` files completely empty (zero bytes).  Putting code in them is possible but I always find that it leads to a lot of unintuitive behavior.

Reply:
Is this resolved?